frontend/javascripts/oxalis/view/settings/dataset_settings_view.js

```diff
@@ -171,6 +175,7 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps> {
   getSegmentationPanel() {
     return (
       <Panel header="Segmentation" key="2">
+        <div style={{ overflow: "auto" }}>{this.getFindDataButton("segmentation", false)}</div>
```

Comment:
In case of a volume tracing, the segmentation layer will have a different name. Please use the name variable of the segmentation layer here. :)